pkg/metrics/metrics.go

```diff
@@ -107,21 +107,34 @@ The label 'result' can have the following values:
 		}, []string{"cluster_queue"},
 	)
 
+	EvictedWorkloads = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "evicted_workloads",
```

Comment:
If this is a counter metric add the _total suffix, following the convention: https://prometheus.io/docs/practices/naming/